dio/test/upload_test.dart

```diff
@@ -97,6 +97,35 @@ void main() {
     },
     testOn: 'vm',
   );
+
+  test('send progress', () async {
+    final data = <Uint8List>[
+      Uint8List.fromList([10, 1]),
+      Uint8List.fromList([1, 4]),
+      Uint8List.fromList([5, 1]),
+      Uint8List.fromList([1, 1]),
+      Uint8List.fromList([2, 4]),
+    ];
+    final stream = Stream.fromIterable(data);
+    final List<int> collected = [];
+    final _ = await dio.put(
+      '/put',
+      data: stream,
+      onSendProgress: (a, b) {
+        expect(b, 10);
```

Comment:
Same as others. Once we modified the bytes, 10 becomes invalid and no one knows what it means.